pinot-controller/src/test/java/org/apache/pinot/controller/api/resources/TableSizeReaderTest.java

```diff
@@ -224,21 +223,21 @@ public void run() {
   }
 
   @Test
-  public void testNoSuchTable()
-      throws InvalidConfigException {
-    TableSizeReader reader = new TableSizeReader(executor, connectionManager, _controllerMetrics, helix);
+  public void testNoSuchTable() throws Exception {
```

Comment:
Please reformat these files using the Pinot Style

Reply:
Done.